help/en/html/tools/logixng/reference/chapter5.shtml

```diff
@@ -810,28 +946,115 @@
         "506"></a>
       </div>
 
-      <h5>Log local variables</h5>
+      <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
+
+      <h4>Log local variables</h4>
 
       <p>See <a href="chapter8.shtml#LogLocalVariables">Chapter 8 - Debugging local variables</a>
       for details.</p>
 
-      <h5>Logix</h5>
+      <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
+
+      <h4>Logix</h4>
 
       <p>LogixNG has an action called Logix that emulates how JMRI Logix works. The action Logix
       has one expression and a number of boolean actions. A boolean action is an action that takes
       a boolean value (true/false) and then decides what to do. The action Logix was added mainly
       to support import of JMRI Logix to LogixNG.</p>
 
-      <h5>Shutdown</h5>
+      <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
+
+      <h4 id="showDialog">Show dialog<span class="since">since 4.25.8</span></h4>
+
+      <p>The <strong>Show dialog</strong> action displays a window that can be used provide
```

Comment:
English is not my native language and I might be wrong, but I think the word `to` is missing before the last word `provide`.

Reply:
Good catch.